content/api/plugins/after-run-api.md

```diff
@@ -15,8 +15,8 @@ machine on which `cypress run` is called.
 <Alert type="warning">
```

Comment:
I see this warning in 20 places in this branch. Can you move this whole `<Alert>...</Alert>` into a partial and replace it with `::include{file=partials/warning-setup-node-events.md}`?

Reply:
ah yes, good call.

Reply:
Also change `commands in this file` to `commands in this function`